installer/src/main/java/com/teamscale/profiler/installer/Installer.java

```diff
@@ -4,43 +4,70 @@
 import com.teamscale.profiler.installer.steps.InstallAgentFilesStep;
 import com.teamscale.profiler.installer.steps.InstallEtcEnvironmentStep;
 import com.teamscale.profiler.installer.steps.InstallSystemdStep;
-import org.conqat.lib.commons.collections.CollectionUtils;
-import org.conqat.lib.commons.system.SystemUtils;
+import com.teamscale.profiler.installer.steps.InstallWindowsSystemEnvironmentStep;
+import com.teamscale.profiler.installer.utils.TeamscaleUtils;
+import com.teamscale.profiler.installer.windows.IRegistry;
+import com.teamscale.profiler.installer.windows.WindowsRegistry;
+import org.apache.commons.lang3.SystemUtils;
 import picocli.CommandLine;
 
-import java.net.URI;
-import java.net.URISyntaxException;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.List;
+import java.util.function.Supplier;
 
 /** Installs the agent system-globally. */
 public class Installer {
 
-	private static final Path DEFAULT_INSTALL_DIRECTORY = Paths.get("/opt/teamscale-profiler/java");
+	private static final Path DEFAULT_INSTALL_DIRECTORY = windowsOrLinux(
+			() -> Paths.get(System.getenv("ProgramFiles")).resolve("teamscale-profiler/java"),
+			() -> Paths.get("/opt/teamscale-profiler/java")
+	);
+
 	private static final Path DEFAULT_ETC_DIRECTORY = Paths.get("/etc");
 
-	private static final String RERUN_ADVICE = getRerunAdvice();
+	private static final String RERUN_ADVICE = windowsOrLinux(
+			() -> "Try running this installer as Administrator.",
+			() -> "Try running this installer as root, e.g. with sudo."
+	);
+
+	private static final String RESTART_ADVICE = windowsOrLinux(
+			() -> "Please restart Windows to apply all changes.",
+			() -> "In an interactive session, you have to log out and log back in for the changes to take effect."
+	);
 
-	private static String getRerunAdvice() {
-		if (SystemUtils.isWindows()) {
-			return "Try running this installer as Administrator.";
+	private static <T> T windowsOrLinux(Supplier<T> windowsSupplier, Supplier<T> linuxSupplier) {
+		if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
There is currently a mixture of `SystemUtils.isWindows()` and `SystemUtils.IS_OS_WINDOWS`. I think we should try to stick to one version if possible.